jobs/JGLOBAL_OFFLINE_ATMOS_ANALYSIS

```diff
@@ -23,7 +23,7 @@ export APREFIX="${RUN}.t${cyc}z."
 export GPREFIX="${GDUMP}.t${gcyc}z."
 
 RUN="gdas" DUMP="gdas" YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
-  COMIN_OBSDMP:COM_OBSDMP_TMPL \
+  COMIN_OBSDMP:COM_OBSPROC_TMPL \
```

Comment:
I replaced `COM_OBSDMP` with `COM_OBSPROC` in a recent PR, to conform to `COM_model` naming convention. Check for other instances of `COM_OBSDMP` in your scripts and change to the `COM_OBSPROC` equivalent.
```suggestion
  COMIN_OBSPROC:COM_OBSPROC_TMPL \
```